### DISCUSSION TITLE
Option to always paste without formatting?

### DISCUSSION THREAD

**Discussion by grannymaster** [TriliumNext/Trilium#4328] [Q&A]
The title says it all. Is there an option that would ignore formatting? Basically, reversing the current approach.

**Comment by Nriver** [comment]
Paste with `ctrl shift v`

**Reply by grannymaster** [reply]
Yes I know that is an option. I looking if there is a option for Trilium I can change.

**Reply by zadam** [reply]
No, this is not possible.

I'm not even sure if this is something which could be overriden in electron (in a browser very likely no).
